README.md

```diff
@@ -0,0 +1,58 @@
+# CometBFT Knowledge Base
+
+[![Creative Commons Attribution 4.0 International
+License][license-img]][license]
+
+This repository is intended to act as a team [Zettelkasten][zk] for the CometBFT
+core team as an attempt to preserve both knowledge gained and insights gleaned
+by the team - both for the current team, and future people working on the
+project.
+
+It is _always_ a work-in-progress. It is made public in case other people or
+users may find it to be of use/interest.
+
+## Requirements
+
+At present, [Foam] is used for editing and navigating the content - this may
```

Comment:
Should we relax the sentence?

Foam _can_ be used? Some authors are using Foam, which is not a requirement thought.